hanan_shirin_qays/moivesapp/src/Components/MovieItem.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Card, { CardBody } from "react-bootstrap/Card";
+import ReactStars from "react-rating-stars-component";
+
+export default function MovieItem(props) {
+  const baseUrl = "https://image.tmdb.org/t/p/w500/";
+  const nullPhoto =
+    "https://image.tmdb.org/t/p/w500/wwemzKWzjKYJFfCeiB57q3r4Bcm.png";
+
+
+  return (
+    <Card key={props.id} style={{ width: "50%" }}>
+      <div class="card image">
```

Comment:
jsx uses `className` not `class`
```suggestion
      <div className="card image">
```